### ISSUE TITLE
Making the header containing site title and nav smaller

### ISSUE THREAD

**Issue by paaljoachim** [wplemon/gridd#181]
I began experimenting with the header - site title and nav.
The centering of the title looks nice, so I will keep it that way for now.

![Screen Shot 2020-04-28 at 13 18 58](https://user-images.githubusercontent.com/5323259/80481521-e5908a00-8952-11ea-811e-042431b97e1b.png)

--------

Looking at how much space the site title is taking.
![Screen Shot 2020-04-28 at 13 21 15](https://user-images.githubusercontent.com/5323259/80481678-343e2400-8953-11ea-8d52-d08414c48b38.png)

--------

Looking at how much space the navigation is taking.
![Screen Shot 2020-04-28 at 13 22 57](https://user-images.githubusercontent.com/5323259/80481788-6ea7c100-8953-11ea-9467-dd34b728846e.png)

----------

Making the Branding smaller results in this.
<img width="1251" alt="Screen Shot 2020-04-28 at 13 23 38" src="https://user-images.githubusercontent.com/5323259/80481854-867f4500-8953-11ea-87bd-cb757af533ce.png">

Frontend:
![Screen Shot 2020-04-28 at 13 25 19](https://user-images.githubusercontent.com/5323259/80481980-bcbcc480-8953-11ea-8c30-cdb8a7762403.png)

Bottom line is that there navigation background is taking too much space. 
Making the navigation area smaller just pushes it downward and does not change the background area of the nav.

<img width="1281" alt="Screen Shot 2020-04-28 at 13 30 22" src="https://user-images.githubusercontent.com/5323259/80482396-7156e600-8954-11ea-8951-d581a0b4f115.png">

Perhaps the easiest would be to remove the nav background color. As that would likely make it possible to nudge much closer to the site title without overlapping it.

**Comment by aristath** [comment]
Looks to me like the problem there is that the site-name is extremely large! Actually scratch that, everything is large. And that's not a bad thing, but whitespace should be proportionate to the other elements. 
If the site-name is 70px and the text-size is 23px (which is what I get on my screen-size), then the whitespace between them should accomodate these sizes.
50px is too thin a container to hold a 70px text...
I would suggest you either use `auto` for the height of these columns, or if you want to hardcode a value for them then at least use `em` values so that they adapt depending on the text that contains them (since the text-size changes depending on the viewport)

**Comment by paaljoachim** [comment]
Yeah I agree. The site name is large. I am going by default values here and adjusting through the customizer. I see this css code: font-size: calc(var(--sts,2) * 1em);
Body Font-Size was on 18px and I brought it down to 16px.

The Branding row:
I brought it up from 50px to 70px.
I also tested auto - but that brought in way too much white space. So I took it back to 70px again. Then noticed you mentioned em so I noticed that 3.2em worked well. I still want to bring the nav closer to the site title though.

https://designkurs.com/

**Comment by aristath** [comment]
Decreased the vertical padding of site-branding, should be a lot better now :+1:
